docs/source/tutorials/audio.ipynb

```diff
@@ -54,6 +84,8 @@
    "outputs": [],
    "source": [
     "# Package installation (hidden on docs website).\n",
+    "# Package versions used: tensorflow==2.8.1 numpy==1.21.6 torch==1.11.0 pandas==1.3.2 scikit-learn==1.0.2 \n",
+    "# pathlib==1.0.1 torchaudio==0.11.0 speechbrain==0.5.11 cleanlab==2.0.0 \n",
```

Comment:
One thing to confirm is that the package versions you've listed in the notebooks do not contradict those in docs/requirements.txt.

Can you just quickly confirm this?  If they contract, I'd be inclined to go with the more recent version of the package (updating docs/requirements.txt as necessary).  Be super careful about this though, I purposefully downgraded some package versions as their latest versions caused errors when running the tutorials

Reply:
Some (like numpy) are missing, I am adding them with the most recent version I ran?

Reply:
Are you talking about them being missing from docs/requirements.txt?

docs/requirements.txt is only supposed to list *extra* dependencies that are not already cleanlab dependencies. 
You probably should not be listing versions of cleanlab dependencies here (just like we decided not to list the version of cleanlab).  Cleanlab dependencies are automatically installed with cleanlab, we shouldn't encourage user to separately install other versions of them.

Reply:
Oops ok I'll amend!